Tropos/Models/Temperature.swift

```diff
@@ -13,7 +13,7 @@ func -(lhs: Temperature, rhs: Temperature) -> Temperature {
     return Temperature(fahrenheitValue: lhs.fahrenheitValue - rhs.fahrenheitValue)
 }
 
-@objc class Temperature: NSObject, Printable {
+@objc class Temperature: NSObject {
```

Comment:
Couldn't we still declare conformance to `CustomStringConvertible`? That would let us string-interpolate an instance of `Temperature` in the future.